src/backends/tensorrt/plan_backend.cc

```diff
@@ -414,6 +413,14 @@ PlanBackend::CreateExecutionContexts(
               mn_itr->second, dla_core_id, &eit->second.first,
               &eit->second.second));
 
+          if (dla_core_id != -1) {
```

Comment:
Shouldn't all these logs be in LoadPlan where the engine and runtime are actually created?
Also, you should always print the gpu_device, even when dla is used.

Reply:
The reason we cannot log this inside LoadPlan is because in the dynamic shape we destroy the engine after creation. 
We also do not pass the gpu_id, and model name to LoadPlan().

Reply:
I see, then can just reduce it to a single line. Since it is verbose log it is ok to just always print gpu id and dla id even when dla id is -1